### ISSUE TITLE
Resources tab in Dataflint crashes with application error

### ISSUE THREAD

**Issue by nclaeys** [dataflint/spark#21]
**Describe the bug**
When I open the resources tab it crashes with an application error:
```
Unexpected Application Error!
o.default.hcl is not a function

TypeError: o.default.hcl is not a function
    at g (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:2:656155)
    at FN (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:102:490636)
    at mo (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:2:1933241)
    at xl (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:2:1992892)
    at vc (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:2:1981998)
    at _c (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:2:1981926)
    at mc (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:2:1981789)
    at ac (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:2:1978569)
    at lc (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:2:1978960)
    at zi (http://localhost:46843/history/spark-32040a8cc0d843c28ed2d07d5e892f6f/dataflint/:2:1919552)
```

**Environemnt**
spark verison: 3.5
platform: K8s
dataflint: 0.2.5 -> validated it works on 0.1.6

**To Reproduce**
1. Go to Dataflint UI
2. Click on resources tab

**Expected behavior**
I expect the resources tab to load

**Screenshots**
![Screenshot from 2024-10-31 11-09-49](https://github.com/user-attachments/assets/18b5f418-e3b0-459c-96f2-b20b1d21a789)

**Additional context**
I don't do anything special and see it for all applications.

**Comment by menishmueli** [comment]
Hi @nclaeys, I have a fix ready, will be fixed next version (0.2.6), really sorry for the inconvenience 
You can downgrade to 0.2.3 for the time being

**Comment by nclaeys** [comment]
Thanks for the quick response, yes will downgrade to version 0.2.3 for the time being.
